include/unifex/get_stop_token.hpp

```diff
@@ -43,8 +43,11 @@ namespace _get_stop_token {
           return (StopToken&&) stoken_;
         }
       };
+#if !defined(__GNUC__) || (__GNUC__ < 10)
+      // GCC 10 (and 11) bug ? deduction guide must be declared at namespace scope
       template <typename StopToken>
       _awaiter(StopToken) -> _awaiter<StopToken>;
+#endif
```

Comment:
Bummer. Maybe we should move the `_awaiter` out to namespace scope, then.

Reply:
Should not be in this PR, sorry, I'll remove.
I'll probably propose another PR one for some fixes I made for GCC >= 10 support.